src/components/image-zoom.component.js

```diff
@@ -68,11 +68,22 @@ export class ImageZoom extends Component {
 
     if (this.state.imgZoom) {
       return (
-        <Modal animationType={'fade'} onRequestClose={() => this.closeModal()}>
+        <Modal
+          nativeId="image-zoom-modal"
+          animationType={'fade'}
+          onRequestClose={() => this.closeModal()}
+        >
           <ModalContainer>
-            <StyledPhotoView onTap={() => this.closeModal()} source={uri} />
+            <StyledPhotoView
+              nativeId="image-zoom-photo-view"
+              onTap={() => this.closeModal()}
+              source={uri}
+            />
 
-            <CloseButton onPress={() => this.closeModal()}>
+            <CloseButton
+              nativeId="image-zoom-close-button"
+              onPress={() => this.closeModal()}
```

Comment:
Now that you're here, can you change this for `onPress={this.closeModal}` this is better for perf.